### PR TITLE
feat: add alby budget

### PR DESCRIPTION
**Pull request by Extheoisah**

### CODE REVIEW

File: src/contexts/payment-context.tsx [hunk 1]
```diff
@@ -50,29 +54,41 @@ export const PaymentContextProvider = ({
...
+        const res = await webln.sendPayment(payment_request);
+        if (res instanceof Error) {
+          setError("could not pay with webln");
+          setIsPaymentModalOpen(true);
+          return { error: res, preimage: null };
+        }
+
+        return { error: null, preimage: res.preimage };
+      } catch (error) {
+        setError("could not pay with webln");
```
Comment (Emmanuel-Develops) [inline src/contexts/payment-context.tsx:86]: any reason why we are not showing the payment modal like we are on `line 80`

Comment (Extheoisah) [reply src/contexts/payment-context.tsx:86]: Its an over-sight. will add it

Comment (Extheoisah) [reply src/contexts/payment-context.tsx:86]: fixed here -> https://github.com/bitcoinsearch/chat-btc/pull/52/commits/7b3154b304cbf08c03af22ee048c9ab69ba50c30

**Comment by nully0x** [comment]
This PR resolves #52

**Review by Emmanuel-Develops — APPROVED** [review]

**Pull request merged**